### PR TITLE
chore:Due to the django pin in base.in this package fails to install.

### PR DESCRIPTION
**Pull request by awais786**
Due to the current pin of `django` in base.in this package fails to install in otherservcies with django3.0 tox environment.

It shows following error.
django-config-models 2.1.1 depends on Django<3 and >=1.8

https://github.com/edx/edx-enterprise/pull/1281/checks?check_run_id=2904230945

### CODE REVIEW

**Review by mzulqarnain1 — APPROVED** [review]

**Comment by awais786** [comment]
@jmbowman Please review. I will bump and release.

**Comment by awais786** [comment]
I want to see the warnings in edx-enterprise with django30. but right now build is not running due to the conflicting versions.
edx-enterprise is using `NullBooleanField` so checking those warnings.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
         python-version: ['3.8']
-        toxenv: [django22-drf39, django22-drflatest, quality, docs]
+        toxenv: [django22-drf39, django22-drflatest, quality, docs, django30-drflatest, django31-drflatest]
```
Comment (jmbowman) [inline .github/workflows/ci.yml:19]: This would be slightly cleaner with an entry like `django{22,30,31}-drflatest`

**Review by jmbowman — APPROVED** [review]
One minor suggestion, otherwise 👍

**Pull request merged**